Documentation/Tools/MigrationWindow.md

```diff
@@ -0,0 +1,38 @@
+
+# Migration Window
+
+When SDK components get obsolete and need upgrade, large projects might require an upgrade tool to allow user controlled Migration.
```

Comment:
```suggestion
When SDK components get obsolete and need upgrading, some projects might require an upgrade tool to allow user controlled migration.
```

Reply:
I may also think about rewording this more along the line of:

As the MRTK undergoes changes, some components may be deprecated and replacements will get introduced. Consumers may will need to migrate to those new replacements, and the Migration Window is a tool that helps to automatically migrate a subset of those changes.

Reply:
Or something else I dunno English isn't my first language.

Reply:
i thought it is :O